base_layer/wallet/src/output_manager_service/service.rs

```diff
@@ -797,14 +846,13 @@ where
         );
         // TODO: Include asset metadata here if required
         // We assume that default OutputFeatures and Nop TariScript is used
+        let output_features = OutputFeatures { ..Default::default() };
         let metadata_byte_size = self
             .resources
             .consensus_constants
             .transaction_weight()
             .round_up_metadata_size(
-                OutputFeatures::default().consensus_encode_exact_size() +
-                    script![Nop].consensus_encode_exact_size() +
-                    Covenant::new().consensus_encode_exact_size(),
+                output_features.consensus_encode_exact_size() + script![Nop].consensus_encode_exact_size(),
```

Comment:
Are you sure the covenant size is 0?

Reply:
Yip, this is an error, reverting to the previous code.